lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -494,9 +494,27 @@ def do_3d_projection(self, renderer):
 
         ecs = (_zalpha(self._edgecolor3d, vzs) if self._depthshade else
                self._edgecolor3d)
+
+        # Sort the points based on z coordinates
+        # Performance optimization: Create a sorted index array and reorder
+        # points and point properties according to the index array
+        z_markers_idx = np.argsort(vzs)[::-1]
+
+        # Re-order items
+        vzs = vzs[z_markers_idx]
+        vxs = vxs[z_markers_idx]
+        vys = vys[z_markers_idx]
+        fcs = fcs[z_markers_idx]
+        ecs = ecs[z_markers_idx]
+        vps = np.vstack((vxs, vys)).T
```

Comment:
Why change to `vstack.T` instead of `column_stack`?

Reply:
Both are fine. I've changed to `column_stack`, which seems a bit clearer.

Interestingly, `vstack.T` is marginally faster for > 10k elements:
![grafik](https://user-images.githubusercontent.com/2836374/81505879-e7fbc800-92f2-11ea-868f-8c4589fbffad.png)
But 10ms difference is not the end of the world also for 1e6 elements. Probably all the surrounding code is much slower anyway.